.pre-commit-config.yaml

```
@@ -36,4 +36,4 @@ repos:
     types: [python]
     language: node
     pass_filenames: false
-    additional_dependencies: ["pyright@1.1.322"]
+    additional_dependencies: ["pyright@1.1.335"]
```

```suggestion
    additional_dependencies: ["pyright@1.1.335"]  # try to keep this in sync with .github/workflows/ci.yml
```
while we're at it, might as well make sure you get the reminder no matter where you look